.github/workflows/test-gator.yaml

```diff
@@ -33,7 +33,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        KUBERNETES_VERSION: ["1.29.10", "1.30.6", "1.31.2"]
+        KUBERNETES_VERSION: ["1.30.10", "1.31.6", "1.32.3", "1.33.0"]
```

Comment:
can you update this to the latest patch version for each minor? https://github.com/kubernetes/kubernetes/tags

Reply:
I didnt update to `1.30.11/12` and `1.31.7` because I didnt find these images in https://hub.docker.com/r/kindest/node/tags.